testing/vcs/test_vcs_set_cmap_colorcell.py

```diff
@@ -0,0 +1,10 @@
+import vcs
```

Comment:
no, we have established a policy of one test per feature. It takes 2 seconds at most to run, this will save one second.